modules/restoring-red-hat-quay-standalone.adoc

```diff
@@ -0,0 +1,214 @@
+----
++
+Example output:
++
+[source,terminal]
+----
+You are now connected to database "example-restore-registry-quay-database" as user "postgres".
+----
+
+. Create a `pg_trmg` extension of your Quay database by running the following command:
++
+[source,terminal]
+----
+example-restore-registry-quay-database=# create extension pg_trgm ;
+----
++
+Example output:
++
+[source,terminal]
+----
+CREATE EXTENSION
+----
+
+. Exit the postgres CLI by entering the following command: 
++
+[source,terminal]
+----
+\q
+----
+
+. Set the password for your PostgreSQL deployment by running the following command:
++
+[source,terminal]
+----
+$ psql -h 172.24.10.50 -U <username> -W -d quay < /tmp/quay-backup/quay-backup.sql
+----
++
+Example output:
++
+----
+SET
+SET
+SET
+SET
+SET
+----
+
+. In the `/opt/new-quay-install` directory, print the contents of your `DISTRIBUTED_STORAGE_CONFIG` bundle:
++
+[source,terminal]
+----
+$ cat config.yaml | grep DISTRIBUTED_STORAGE_CONFIG -A10
+----
++
+Example output:
++
+[source,yaml]
+----
+DISTRIBUTED_STORAGE_CONFIG:
+   default:
+       - RadosGWStorage
+       - access_key: <access_key>
+         bucket_name: quay
+         hostname: 172.24.10.50
+         is_secure: false
+         port: 9000
+         secret_key: <secret_key>
+         storage_path: /datastorage/registry
+DISTRIBUTED_STORAGE_DEFAULT_LOCATIONS: []
+----
+
+. Export the `AWS_ACCESS_KEY_ID` by using the `access_key` credential obtained in Step 13:
++
+[source,terminal]
+----
+$ export AWS_ACCESS_KEY_ID=<access_key> 
+----
+
+. Export the `AWS_SECRET_ACCESS_KEY` by using the `secret_key` obtained in Step 13:
++
+[source,terminal]
+----
+$ export AWS_SECRET_ACCESS_KEY=<secret_key>
+----
+
+. Create a new s3 bucket by entering the following command:
++
+[source,terminal]
+----
+$ aws s3 mb --no-verify-ssl --endpoint-url http://172.24.10.50:9000 s3://quay
+----
++
+Example output:
++
+[source,terminal]
+----
+$ make_bucket: quay
+----
+
+. Upload all blobs to the new s3 bucket by entering the following command: 
++
+[source,terminal]
+----
+$ aws s3 sync --no-verify-ssl --endpoint-url http://172.24.10.50:9000 /tmp/quay-backup/blob-backup/. s3://quay/
+----
++
+Example output: 
++
+[source,terminal]
+----
+upload: ../../tmp/quay-backup/blob-backup/datastorage/registry/sha256/50/505edb46ea5d32b5cbe275eb766d960842a52ee77ac225e4dc8abb12f409a30d to s3://quay/datastorage/registry/sha256/50/505edb46ea5d32b5cbe275eb766d960842a52ee77ac225e4dc8abb12f409a30d
+upload: ../../tmp/quay-backup/blob-backup/datastorage/registry/sha256/27/27930dc06c2ee27ac6f543ba0e93640dd21eea458eac47355e8e5989dea087d0 to s3://quay/datastorage/registry/sha256/27/27930dc06c2ee27ac6f543ba0e93640dd21eea458eac47355e8e5989dea087d0
+upload: ../../tmp/quay-backup/blob-backup/datastorage/registry/sha256/8c/8c7daf5e20eee45ffe4b36761c4bb6729fb3ee60d4f588f712989939323110ec to s3://quay/datastorage/registry/sha256/8c/8c7daf5e20eee45ffe4b36761c4bb6729fb3ee60d4f588f712989939323110ec
+...
+----
+
+
```

Comment:
Use the saved container run command from the backup section read back from a file and change any impacted paths to `/opt/new-quay-install` where applicable.